rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/jobs/local/JobExecutorLocal.groovy

```diff
@@ -110,6 +110,22 @@ class JobExecutorLocal implements JobExecutor {
   }
 
   @Override
+  BakeStatus getJob(String jobId) {
```

Comment:
Since the `BakeStatus` object returned doesn't contain anything meaningful beyond the `jobId` that was passed in as an argument, this would probably make more sense (and be easier to read) as: `boolean jobExists(String jobId)`.